db/install.xml

```diff
@@ -28,7 +28,7 @@
         <FIELD NAME="versionhash" TYPE="char" LENGTH="256" NOTNULL="true" SEQUENCE="false" COMMENT="All version hash"/>
         <FIELD NAME="datasize" TYPE="int" LENGTH="11" NOTNULL="true" SEQUENCE="false" COMMENT="Size of the profile data (in bytes)"/>
         <FIELD NAME="numsamples" TYPE="int" LENGTH="11" NOTNULL="true" SEQUENCE="false" COMMENT="Number of samples taken"/>
-        <FIELD NAME="flamedatad3" TYPE="text" NOTNULL="false" SEQUENCE="false" COMMENT="JSON structure compatible with d3-flame-graph."/>
+        <FIELD NAME="flamedatad3" TYPE="binary" NOTNULL="false" SEQUENCE="false" COMMENT="compressed JSON structure compatible with d3-flame-graph."/>
```

Comment:
Probably a capital to start off the comment.
```suggestion
        <FIELD NAME="flamedatad3" TYPE="binary" NOTNULL="false" SEQUENCE="false" COMMENT="Compressed JSON structure compatible with d3-flame-graph."/>
```

Reply:
Done